### ISSUE TITLE
git stash support

### ISSUE THREAD

**Issue by Gonkers** [github/VisualStudio#569]
Would you consider adding [git stash](https://git-scm.com/docs/git-stash) support? I find it pretty useful in my work flow. Thanks.

**Comment by grokys** [comment]
Yes, it's something I really miss personally, and something that's notably lacking from Visual Studio's git support. We're currently waiting to see if Microsoft has plans to add stash support as we don't want to duplicate their features. If not, then it's definitely something we'd consider adding ourselves.

**Comment by yannduran** [comment]
+1

**Comment by adrianimajion** [comment]
+1 on this issue as well. This was opened in September 2016, since Github is now part of Microsoft can we cross our fingers on a resolution sometime soon?

**Comment by getuliojr** [comment]
2 years has passed and still we need sourcetree or prompt for that !!!

**Comment by yannduran** [comment]
Even though it's pretty basic functionality to add, it's **still** not _baked in_, so I've been using [this extension](https://marketplace.visualstudio.com/items?itemName=vii.GitStashExtension) for a while now.

The developer seems pretty responsive.
